### PR TITLE
Updated changelog for 0.15

### PR DESCRIPTION
**Pull request by meichstedt**

### CODE REVIEW

**Comment by meichstedt** [comment]
@kolloch I made some adjustments and squashed the commits. Can you please take a quick look?

File: changelog.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+
+#### Integration of Mesos Fetcher Cache
+
+The v2 REST API was extended to support the Mesos fetcher cache. This allows users to configure a list of resource URIs
+that will be copied into the task sandbox prior to running the task, from either a local or external location. For
+details on the fetcher cache's capabilities, please see the fetcher cache [documentation](http://mesos.apache.org/documentation/latest/fetcher/).
+
+#### Migration from Marathon version 0.5 removed
+
+It is no longer possible to migrate from Marathon version 0.5.
```
Comment (kolloch) [inline changelog.md]: Or 0.7.

Comment (kolloch) [inline changelog.md]: Maybe add a note that says that you need to upgrade to intermediary versions as is anyway recommended.

**Comment by meichstedt** [comment]
I addressed your comments and added 2 missing fixes to the list.

**Comment by kolloch** [comment]
:rocket:

**Pull request merged**